rusk/src/lib/node/rusk.rs

```diff
@@ -740,7 +753,7 @@ fn execute(
         .call::<_, ()>(
             TRANSFER_CONTRACT,
             "refund",
-            &receipt.gas_spent,
+            &(receipt.gas_spent, deployed_id),
```

Comment:
I believe we should pass the tx error here, otherwise we don't have the error propagated in caso of deployment error related to the gas limit